js/app/Messages.js

```diff
@@ -1,6 +1,32 @@
 'use strict';
 
 var Polyglot = require( 'node-polyglot' ),
-	i18n = require( '../i18n.json' );
+	i18n;
 
-module.exports = new Polyglot( { phrases: i18n[ 'de' ] } );
+var getParam = function getUrlParameter( sParam ) {
```

Comment:
I thought JS would have a built-in thing to deal with URLs but google didn't spit anything out :/